app/server/src/features/accounts/resolvers.ts

```diff
@@ -3,9 +3,11 @@ import { connectionFromArray, fromGlobalId } from 'graphql-relay';
 import { Resolvers, toGlobalId, runMutation } from '@local/features/utils';
 import { CookieSerializeOptions } from 'fastify-cookie';
 import * as User from './methods';
+import { eventNames } from 'node:process';
```

Comment:
I don't think this is needed? Probably an accident from pressing tab and vscode auto importing.